runner/ansible/roles/1.3.7/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '1.3.7'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    if [[ $(crm_node -l | wc -l) != "2" ]]; then
+      echo passed
```

Comment:
I would prefer to remove this `passed/failed` echos and use the `rc` to check the result.
This way, we could remove the expected value `passed`, which doesn't provide any utility.
We could put an empty string in the vars file, or remove it all together, as this test doesn't have a changeable expected value.
(I would vote for removing)